ui/src/reducers/authors.js

```diff
@@ -19,6 +21,7 @@ import {
 export const initialState = Map({
   publicationSelection: Set(),
   isAssignDrawerVisible: false,
+  publicationSelectionClaimed: Map({}),
```

Comment:
Can this be a `Set`? I don't see why it's different from the `publicationSelection`

Reply:
cause there is also `claimed` value, and each claimed has to be matched with corresponding paperId